contracts/contracts/WrappedNCGToken.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract WrappedNCGToken is ERC20, Ownable {
+    event Burn(address indexed _sender, bytes32 indexed _to, uint256 amount);
+
+    constructor() public ERC20("Wrapped NCG Token", "wNCG") {}
```

Comment:
```suggestion
    constructor() public ERC20("Wrapped NCG", "wNCG") {}
```

Why don't we remove `Token` like [WBTC](https://github.com/WrappedBTC/bitcoin-token-smart-contracts/blob/master/ethereumV2/contracts/token/WBTC.sol#L11)?